python-lib/lal/tokenizers/language_dict.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+"""Module with constants defining the language support of underlying NLP libraries"""
```

Comment:
I'd prefer to keep everything related to NER in one place, so how about creating a directory:

`./lal/classifiers/ner`
and put there text_classifier.py
`classifiers/text_classifier.py` and everything in ./tokenizers

Reply:
Makes very much sense, will do that.

Reply:
Changed in 8c49f72